src/logger.cpp

```diff
@@ -276,8 +248,6 @@ void Logger::appendLog(LogLevel level, const QString &message, bool newline)
  */
 void Logger::directLog(LogLevel level, const QString &message, bool newline)
```

Comment:
Hi @jsbackus,

Could you tell me with this one?

Do you remember what was the reason of differentiating between `directLog` and `appendLog`?  
Were there any performance issues with logs causing these optimizations?